Felienne/hedy
# [BUG FIX] Fixes incorrect quiz translation for English

**Description**
In this PR we make a small content translation fix where question 8 and 9 of level 7 of the English quiz contained some Dutch string. We manually translate this in the YAML file and hope that Weblate is still happy after this, it should be as English is the main language. Meaning that this fix should automatically be distributed to all non-translated question 8 and 9 level 7 for all languages.

**Fixes**
This PR fixes #2800.

**How to test**
Start the quiz in English on level 7. Verify that question 8 and 9 contain only English string instead of a few Dutch ones.
